src/utils/input.ts

```diff
@@ -13,7 +14,7 @@ import TafsirInfo from 'types/TafsirInfo';
  */
 export const generateSelectOptions = (options: string[]): SelectOption[] =>
   options.map((option) => ({
-    label: capitalize(option),
+    label: startCase(option),
```

Comment:
startCase will convert "MuslimWordLeague" -> "Muslim Word League"

Reply:
this should happen before passing the input